src/stream/src/executor/barrier_align.rs

```diff
@@ -80,9 +80,9 @@ impl BarrierAligner {
         loop {
```

Comment:
Is this old `BarrierAligner` still used? IIRC, only Lookup uses it besides HashJoin. If so, we may directly remove this.

Reply:
Lookup now has its own barrier aligner.

Reply:
Yes, the v2 `UnionExecutor` depends on it. We can refactor it in future PR.